src/common-ssh/guac_ssh.h

```diff
@@ -76,10 +76,10 @@ void guac_common_ssh_uninit();
 /**
  * Connects to the SSH server running at the given hostname and port, and
  * authenticates as the given user. If an error occurs while connecting or
- * authenticating, the Guacamole client will automatically and fatally abort.
- * The user object provided must eventually be explicitly destroyed, but should
- * not be destroyed until this session is destroyed, assuming the session is
- * successfully created.
+ * authenticating, the Guacamole client will only automatically and fatally
+ * abort if the abort_on_error parameter is set to 1 and errors will always
```

Comment:
As mentioned earlier, requiring `1` is odd for C. Better for this to simply be non-zero. Other than that, still looking good.